packages/x-components/src/__stubs__/next-queries-stubs.factory.ts

```diff
@@ -4,9 +4,7 @@ import { NextQuery } from '@empathyco/x-types';
  * Creates a {@link @empathyco/x-types#NextQuery | next queries} stub.
  *
  * @param amount - Number of stubbed next queries to create.
```

Comment:
Be careful changing lines that are not needed to be changed.

Reply:
Are you referring to the spacing between lines?
We deleted them as suggested by the linter, is there a problem with that ESLint configuration?